fix: Update reference document regarding images (#220)
citynetwork/docs: docs/reference/images/index.md

* `os_version=${VERSION_ID}`: operating system version, such as `os_version=22.04`

[Other properties](https://docs.openstack.org/glance/latest/admin/useful-image-properties.html) may also be set on individual images.
For instance, the `image_original_user` property reflects the username of the default non-root user of the corresponding operating system -- and that piece of information may come in handy in various automation scenarios.
So, to find out the username of the default non-root user in the `Ubuntu 22.04 Jammy Jellyfish x86_64` image, you can type the following:

```console

> $ openstack image show -f json "Ubuntu 22.04 Jammy Jellyfish x86_64" \

Reviewer: In case you have python (you usually have out of the box) and don't want to install third-party depndencies (like jq), you can also do it like that:

`python3 -c "print($(openstack image show 'Ubuntu 22.04 Jammy Jellyfish x86_64'  -c properties -f value)['image_original_user'])"`

Author: Yes, that works too. I am wondering, though, if the usage of `jq` "feels" more natural to most users -- at least compared to resorting to Python?

    | jq '.properties.image_original_user'
"ubuntu"
```

## Image updates and automation

Images in {{brand}} are updated regularly, and that is something you can deduce from the `replace_frequency` property.
See, for example, the value of this property in the `Ubuntu 22.04 Jammy Jellyfish x86_64` image:

```console
$ openstack image show -f json "Ubuntu 22.04 Jammy Jellyfish x86_64" \
    | jq '.properties.replace_frequency'
"monthly"
```

Per [the SCS reference](https://docs.scs.community/standards/scs-0102-v1-image-metadata/#image-updating), `monthly` here means that the image is replaced *at least once* per month.
Newer images have operating systems with all the latest package updates and security fixes.
Whenever a fresh version of an image is produced and made available, it has the exact same name as the previous version but a *different* UUID.

> That is why in Ansible Playbooks, Heat templates, Terraform configurations, or your automation in general, you should refrain from using UUIDs and refer to images by name instead.

Reviewer: Maybe it's worth to mention how to get a build date as well?

Author: Yes, I think it is :)

* `os_version=${VERSION_ID}`: operating system version, such as `os_version=22.04`

[Other properties](https://docs.openstack.org/glance/latest/admin/useful-image-properties.html) may also be set on individual images.
For instance, the `image_original_user` property reflects the username of the default non-root user of the corresponding operating system -- and that piece of information may come in handy in various automation scenarios.
So, to find out the username of the default non-root user in the `Ubuntu 22.04 Jammy Jellyfish x86_64` image, you can type the following:

```console

> $ openstack image show "Ubuntu 22.04 Jammy Jellyfish x86_64" \
>     -c properties -f json | jq '.[].image_original_user'
> 
> "ubuntu"

Reviewer 2: The combination of `-c <field>` with `-f json` strikes me as odd. I'd suggest: 
```suggestion
$ openstack image show -f json "Ubuntu 22.04 Jammy Jellyfish x86_64" \
    | jq '.properties.image_original_user'
"ubuntu"
```

Author: Done!

```

> 
> And what about the same piece of information from the `Debian 11 Bullseye x86_64` image?
> 
> ```console
> openstack image show "Debian 11 Bullseye x86_64" \
>     -c properties -f json | jq '.[].image_original_user'
> 
> "debian"
> ```

Reviewer 2: This is reference documentation, where one example does suffice. Thus, I'd remove this.
```suggestion
```

Author: Aha, yes.

## Image updates and automation

Images in {{brand}} are updated regularly, and that is something you can deduce from the `replace_frequency` property.
See, for example, the value of this property in the `Ubuntu 22.04 Jammy Jellyfish x86_64` image:

```console
$ openstack image show "Ubuntu 22.04 Jammy Jellyfish x86_64" \
    -c properties -f json | jq '.[].replace_frequency'

> "monthly"

Reviewer 2: This needs to clarify that per [the SCS reference](https://docs.scs.community/standards/scs-0102-v1-image-metadata/#image-updating), a frequency of `monthly` means that the image is replaced _at least_ once a month, not _no more frequently than_ once a month.

Author: Yes, done.

```

Newer images have operating systems with all the latest package updates and security fixes.
Whenever a fresh version of an image is produced and made available, it has the exact same name as the previous version but a different UUID.

> That is why in Absible Playbooks, Heat templates, Terraform configurations, or your automation in general, you should be shying away from using UUIDs and referring to images by name instead.

Reviewer 2: Typo, and some room for condensation:
```suggestion
That is why in Ansible Playbooks, Heat templates, Terraform configurations, or your automation in general, you should refrain from using UUIDs, and refer to images by name instead.
```

Author: Wow, thanks for spotting that (and I condensed too -- omitting that last coma).
